src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEvent.cs

```diff
@@ -11,15 +11,20 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     public class DiagnosticEvent : TableEntity
     {
+        internal const string CurrentEventVersion = "2024-05-01";
```

Comment:
"20240501" anyone ?

Reply:
I like the azure api version model

https://learn.microsoft.com/en-us/rest/api/searchmanagement/management-api-versions#stable-versions